indigo_api/migrations/0129_migrate_refs_uris.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-04-22 12:39
+from __future__ import unicode_literals
+
+from lxml import etree
+
+from django.db import migrations
+
+from cobalt import Act, FrbrUri
+from cobalt.uri import FRBR_URI_RE
+
+
+def new_frbr_uri(uri, forward):
+    """ Sets prefix on uri:
+        'akn' if forward is True, None if it's False.
+    """
+    if not isinstance(uri, FrbrUri):
+        uri = FrbrUri.parse(uri)
+    uri.prefix = 'akn' if forward else None
+    return uri.work_uri()
+
+
+def handle_refs(document, forward):
+    """ Finds hrefs in documents, of the form:
+
+        href="/za/act/2012/22">Act no 22 of 2012
+        href="/akn/za/act/2012/22">Act no 22 of 2012
+
+        For each of these,
+        if `forward` is True, it adds the /akn prefix
+        if `forward` is False, it removes it
+
+    """
+    root = etree.fromstring(document.document_xml)
+    path = etree.XPath("//*[starts-with(@href, '/')]")
```

Comment:
This definitely needs to be namespace aware. Otherwise we're potentially modifying XML that we have no understanding of.